### ISSUE TITLE
[Bug]: `shopify app versions list` formatting issues

### ISSUE THREAD

**Issue by hannachen** [Shopify/cli#2281]
### Please confirm that you have:

- [X] Searched [existing issues](.) to see if your issue is a duplicate. (If you’ve found a duplicate issue, feel free to add additional information in a comment on it.)
- [X] Reproduced the issue in [the latest CLI version](https://www.npmjs.com/package/@shopify/cli).

### In which of these areas are you experiencing a problem?

Extension

### Expected behavior

When I run `shopify app versions list`, the closing parenthesis should be present even on narrow screens.

### Actual behavior

Looks like this instead:
<img width="1035" alt="Screenshot 2023-06-30 at 1 58 46 PM" src="https://github.com/Shopify/cli/assets/2709526/a1cf28e1-9a4f-4f52-a83c-1482bfb99551">

### Verbose output

```shell
N/A
```

### Reproduction steps

1. Create an app with many versions
2. Use the CLI to run `npm run shopify app versions list` from the app project folder

### Operating System

Mac OS Ventura 13.4

### Shopify CLI version (check your project's `package.json` if you're not sure)

Nightly

**Comment by ani-shopify** [comment]
@isaacroldan I used the nightly version and still ran into the same issue

![image](https://github.com/Shopify/cli/assets/99370588/9caaf2c3-5145-4123-84e9-c6a508546f48)

It does seem to be related to the space as further zooming in the string gets moved to a new line:

![image](https://github.com/Shopify/cli/assets/99370588/849aa728-d96f-4d70-ab94-e7feafd68fc0)


Are we truncating the string by any chance?

**Comment by isaacroldan** [comment]
We are not truncating before rendering the table, but the table might do its own things :/ 
What we want is the second screenshot right? is hard to render this much information in a small terminal
I'll try to take a look to the table implementation

**Comment by ani-shopify** [comment]
> What we want is the second screenshot right? 

Yes, exactly. Would be nice to get the string move to the next line instead of swallowing it.
